portal/src/main/scala/cromwell/pipeline/service/GitClient.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.pipeline.service
+
+import java.io.File
+
+import cromwell.pipeline.datastorage.dto.Project
+
+trait GitClient {
+  def updateFile(project: Project, file: File)
+  def updateListOfFiles(project: Project, files: List[File])
+  def createRepository(project: Project)
+  def getListOfFiles(project: Project, file: File)
```

Comment:
Please, specify the return types. Without it, the compiler thinks that the return type is `Unit`.